infoschema/perfschema/tables_test.go

```diff
@@ -250,6 +251,63 @@ func (s *testTableSuite) TestStmtSummaryTable(c *C) {
 		max_prewrite_regions, avg_affected_rows, query_sample_text, plan
 		from performance_schema.events_statements_summary_by_digest`,
 	).Check(testkit.Rows())
+
+	// Create a new session to test
+	tk = testkit.NewTestKitWithInit(c, s.store)
+
+	tk.MustExec("set global tidb_enable_stmt_summary = on")
+	tk.MustExec("set global tidb_stmt_summary_history_size = 100")
```

Comment:
Defer set them back, otherwise it'll affect other test cases.

Reply:
I have tried to set them back.. but it really confused me that the tests are still failed T_T

Reply:
I have tried to set them back.. but it really confused me that the tests are still failed T_T